<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CForecastBtn</class>
 <widget class="QWidget" name="CForecastBtn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>72</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="weekLab">
     <property name="text">
      <string>星期</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="weatherLab">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="tempLab">
     <property name="text">
      <string>0°C~0°C</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
